osh/cmd_parse.py

```diff
@@ -2182,7 +2198,8 @@ def _ParseAndOr(self):
       if self.c_id not in (Id.Op_DPipe, Id.Op_DAmp):
         break
 
-    node = command.AndOr(ops, children)
+    assert len(op_toks) >= 1, "There must be at least 1 operator token"
+    node = command.AndOr(op_toks[0], ops, op_toks, children)
     node.spids = op_spids
```

Comment:
It seems like we could remove op_spids and do something like:

```
node.spids = [tok.span_id for tok in op_toks]
```

Although since we're going to get rid of all the `node.spids` soon anyway, maybe it doesn't matter